### ISSUE TITLE
False positive #1? 

### ISSUE THREAD

**Issue by geoffrk00** [microsoft/accessibility-insights-web#6100]
https://webcompliance.trafficmanager.net/ScanResultsApi/GetAccessibilityScanConsolidatedReport?siteId=8172d1b3-b36c-e711-80c2-0004ffc0fa4b

WCP is showing an error on 
https://learn.microsoft.com/en-us/
duplicate-id-aria: Ensures every id attribute value used in ARIA and in labels is unique

<div class="dropdown-menu" id="theme-menu" role="menu">

This ID is not duplicated 
Running AI4W locally on this page does not show this error

**Comment by jlperkins** [comment]
Agreed; running automated checks against the url that reportedly has this failure, does not produce the same failed result.  Thank you for bringing this to our attention; the team will triage this issue and address it.
